fusor-ember-cli/app/styles/wizard.scss

```diff
@@ -254,6 +254,6 @@
   padding-top: 20px;
 }
 
-.sidebar-pf.sidebar-pf-left {
-  min-height: 850px;  /* to expand left bar to be gray -- temp fix rather than calculating page height  in jQuery*/
-}
```

Comment:
@rhamilto, @knowncitizen, @jeff-phillips-18 Previously, I did this, which is now deleted.